lesson3/app/src/main/java/io/wolfd/mobileproto/lesson3/SettingsFragment.java

```diff
@@ -10,12 +10,15 @@
 import android.view.ViewGroup;
 
 public class SettingsFragment extends Fragment {
+    // What does Nullable do? I actually don't know.
```

Comment:
`@Nullable` on a method means that it can potentially return null. On a parameter it means that you can pass in null and it should be able to handle it. These were just added by intellij though.